pandas/core/frame.py

```diff
@@ -6449,31 +6451,31 @@ def join(self, other, on=None, how='left', lsuffix='', rsuffix='',
         other : DataFrame, Series with name field set, or list of DataFrame
```

Comment:
Do you mind removing the `with name field set` from the types, and have it in the description instead?